### PR TITLE
We have iOS-sources

### PR DESCRIPTION
**Pull request by MariusBluem**
Signed-off-by: Marius Blüm <marius@lineone.io>

### CODE REVIEW

File: install-instructions.php [hunk 1]
```diff
@@ -167,7 +167,7 @@
 									</a>
 								</div>
 							</div>
-							<p><a class="hyperlink" href="https://github.com/nextcloud/android"><?php echo $l->t('Find Android Sources here</a>, <a class="hyperlink" href="https://download.nextcloud.com/android">direct APK download here</a> and <a class="hyperlink" href="https://github.com/nextcloud/windows-uwp">Windows Mobile sources here</a>.');?></p>
+							<p> <a class="hyperlink" href="https://github.com/nextcloud/ios"><?php echo $l->t('Find iOS Sources here</a>, <a class="hyperlink" href="https://github.com/nextcloud/android"><?php echo $l->t('Android Sources here</a>, <a class="hyperlink" href="https://download.nextcloud.com/android">direct APK download here</a> and <a class="hyperlink" href="https://github.com/nextcloud/windows-uwp">Windows Mobile sources here</a>.');?></p>
```
Comment (MorrisJobke) [inline install-instructions.php]: Sadly there is a syntax error. This is too much `<?php echo $l->t('` if I see this correctly.

Comment (MariusBluem) [inline install-instructions.php]: Sorry for that 😢

Comment (jospoortvliet) [inline install-instructions.php]: Yeah, I see it too, just fixed, merged and pushed to live. Install page should work again as soon as caches are invalidated.

Comment (jospoortvliet) [inline install-instructions.php]: (can take 30 minutes apparently, varnish :( )

Comment (NinaNC) [inline install-instructions.php]: Sorry, I didn't see the syntax error. Shouldn't have merged it, I will wait for another review next time.

Comment (LukasReschke) [inline install-instructions.php]: Manually forced invalidation of all caches.

Comment (MariusBluem) [inline install-instructions.php]: > Manually forced invalidation of all caches.

👍

Comment (MorrisJobke) [inline install-instructions.php]: Works again

Comment (blizzz) [inline install-instructions.php]: Is it feasible to have a linter run against php files?

Comment (LukasReschke) [inline install-instructions.php]: Linter at https://github.com/nextcloud/nextcloud.com/pull/518

**Pull request merged**